_posts/francois_gerthoffert/2019-03-03-ZenCrepes-GitHub-Agile-Analytics-made-easy.md

```diff
@@ -0,0 +1,301 @@
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+Approximately a year ago, our team transitioned away from Jira to GitHub for our ticketing system, bringing our source code and project management in the same place. A few months back, I wrote a [blog post detailing this transition](https://softeng.oicr.on.ca/francois_gerthoffert/2018/10/02/Transitioning-away-from-Jira-Lessons-learnt/) and another one giving some [insights about some Agile metrics that could be used for team](https://softeng.oicr.on.ca/francois_gerthoffert/2018/10/20/Agile-another-take-on-estimates/) management, in particular in the context of research. 
+
+Both articles contained screenshots from an “unknown tool” and now is the right time to start introducing ZenCrepes (bonus points for those who guess where the name is coming from), a personal project I’ve been spending evenings on for the past few months to provide Agile analytics and management across GitHub organizations & repositories.
+
+# Why ZenCrepes?
+
+In an ideal (naive?) world, one product would have one source code repository and one single team, dedicated to this single product, and GitHub would be an awesome platform for both source control and agile management. 
+
+But more and more agile teams span multiple repositories or even multiple organizations. In our case, our scrum teams are frequently using two organizations, one for [Overture.bio](https://github.com/overture-stack/) containing our reusable components used across multiple projects and one for each of our projects such as [ICGC-DCC](https://github.com/icgc-dcc/), [Kids-First](https://github.com/kids-first), [HCMI](https://github.com/nci-hcmi-catalog/)... 
+
+As soon as you begin operating on more than one repository, there is a lack of tools to facilitate management at an organization level or even across organizations. GitHub Projects at an organization level are limited to 5 repositories per project, labels and milestones are unique per repositories... 
+
+ZenCrepes was created to address this limitation, by providing the missing cross-org & cross-repos view.
+
+# Key Features
+
+It focuses on three primary objectives:
+* __Report and search__: Quickly find relevant issues based on selected criteria (faceted search). For example, “List open defects assigned to John or Max in rock or paper repos” or “Display the team's overall velocity on paper and scissor repos”.
+* __Scrum operation__: Provide one scrum view to identify the amount of work left in a sprint, estimate completion based on past velocity, review repartition of open issues (by repo, by labels, by assignees).
+* __Consistency__: Ensure labels and milestones are consistent across multiple organizations and repositories, clean-up when necessary.
+
+
+# Report and search
+
+For those familiar with our portals, no need to search long to understand where the inspiration is coming from for the “Issues” view (no pie charts though!).
```

Comment:
As an insider, I enjoy the inside reference :).  But maybe new readers will not get it?  You could maybe link the word **portals** to [https://dcc.icgc.org/search](https://dcc.icgc.org/search).  Just a suggestion!